doc/whats-new.rst

```diff
@@ -25,6 +25,10 @@ Breaking changes
   merges will now succeed in cases that previously raised
   ``xarray.MergeError``. Set ``compat='broadcast_equals'`` to restore the
   previous default.
+- Pickling an xarray object based on the dask backend, or reading its
+  :py:meth:`values` property, won't automatically convert the backend to numpy
```

Comment:
Instead of "convert the backend to numpy" let's say "convert the array from dask to numpy"

Somewhat confusingly, we use the name `backends` in xarray to refer to IO methods (e.g., netCDF libraries), not dask vs numpy.